store/modules/events.ts

```diff
@@ -134,6 +141,11 @@ export default {
 
       noFilterFound && state[filter].push(filterValue)
     },
+    updateFilterSet (state, payload) {
```

Comment:
We only have test for the getter function `filteredEvents`. We don't have yet test for the mutations functions and `updateFilterSet` is one of them. I think it could be better to create an issue to improve the code quality by adding tests for all the mutation function instead of creating now just one test for `updateFilterSet`. What do you think?

Reply:
Done [here](https://github.com/Qiskit/qiskit.org/pull/1054)